include/vcpkg/base/messages.h

```diff
@@ -751,6 +753,10 @@ namespace vcpkg
     DECLARE_MESSAGE(ExpectedPortName, (), "", "expected a port name here");
     DECLARE_MESSAGE(ExpectedTripletName, (), "", "expected a triplet name here");
     DECLARE_MESSAGE(ExtendedDocumenationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
+    DECLARE_MESSAGE(FailedToExtract,
+                    (msg::path, msg::error),
+                    "'{error}' is the error message.",
+                    "Failed to extract '{path}' with message '{error}'.");
```

Comment:
It seems like it would be better to just have a "Failed to extract \"{path}\":" and append the message to the end outside of the formatting layer.